ReactWindows/ReactNative/Views/Image/ReactImageManager.cs

```diff
@@ -17,14 +23,17 @@ namespace ReactNative.Views.Image
     /// <summary>
     /// The view manager responsible for rendering native images.
     /// </summary>
-    public class ReactImageManager : SimpleViewManager<Border>
+    public class ReactImageManager : BaseViewManager<Border, ReactImageShadowNode>
     {
         private readonly Dictionary<int, SerialDisposable> _disposables =
             new Dictionary<int, SerialDisposable>();
 
         private readonly Dictionary<int, List<KeyValuePair<string, double>>> _imageSources =
             new Dictionary<int, List<KeyValuePair<string, double>>>();
 
+        private Color? _tintColor;
```

Comment:
Good to know. 
I'm using a shadow node, but I know of no way to get the shadow node data within the view manager. Since resize could require a re-render, and size change doesn't trigger UpdateExtraData, I couldn't get the shadow props.

Might go the dictionary route if there's no solution to that.